examples/dashboards/platform_engineer.json

```diff
@@ -750,7 +750,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "count(gatewayapi_gateway_info unless (gatewayapi_gateway_info{exported_namespace=~\"${gateway_namespace}\"} * on(name, exported_namespace, instance) group_left gatewayapi_gateway_status{exported_namespace=~\"${gateway_namespace}\",type=\"Programmed\"})\nor ignoring(type)\ngatewayapi_gateway_info unless (gatewayapi_gateway_info{exported_namespace=~\"${gateway_namespace}\"} * on(name, exported_namespace, instance) group_left gatewayapi_gateway_status{exported_namespace=~\"${gateway_namespace}\",type=\"Accepted\"}))",
```

Comment:
Could you explain a bit more about how the <1 is replacing the unless. I cant work it out in my head. Unless is like complement so before it was counting gatewayapi_gateway_info without gatewayapi_gateway_info(.....) so now we are counting only the timeseries we were originally not wanting

Reply:
Before, I set up the query to include all gateways (`gatewayapi_gateway_info`), unless the status of Programmed was true ,
or all gateways unless unless the status of Accepted was True,
then count all the results.

This wasn't working as expected, so i changed it look for gateways where the status of Programmed was not True (i.e. has a value of 0, which is <1). I could have used an `== 0` or `< 1` check here I think, but I wanted to make sure any value < 1 got included.

```promql
gatewayapi_gateway_status{exported_namespace=~\"${gateway_namespace}\",type="Programmed"}
    < 1
```

Same for Accepted.
The `or ignoring(type)` ensures that in case a gateway is *not* Programmed and *not* Accepted, it doesn't get included twice.
Then all that is wrapped in a count.

Reply:
Ahhh ok im with you now makes sense